### PR TITLE
YALB-458: Add Integration Environment documentation

### PR DESCRIPTION
**Pull request by dens0ne**
### Description of work
- Adds documentation that describes how to manually push content to the "canary" site.

### Functional testing steps:
- [ ] Feel free to open a test branch to follow the documentation, and confirm it makes sense.

### CODE REVIEW

**Review by nJim — APPROVED** [review]
Approved. This was the right level of effort for this ticket. This is the first steps towards the real CI/CD system and simply addresses, how we get changes to the integration site. Mechanically, this is POC for the next tickets.

I greatly appreciate the detailed instructions. In the near future, these instructions may be removed from this README as CircleCi will be handling all of this work. We should still keep a copy handy as this is useful information for anyone who want to test work in this way.

I completed my functional test by creating a change in the yalesites_profile:
https://github.com/yalesites-org/yalesites_profile/commit/f088d37a45e6a927d3f27c2f7fbc366bf7ec8026

And then pushing the changes to this integration site:
https://yalb111-yale-integration-env.pantheonsite.

Following these steps gave me a place to see my site title change. This works great and serves as a good place to move forward to automating the process.

I will check with Vincent to see if he wants to weigh in on this work before we merge it.

File: README.md [hunk 1]
```diff
@@ -4,3 +4,21 @@ The YaleSites installation profile combines a suite of modules, themes, and pre-
...
+- (If creating a multidev) Go to the [Pantheon dashboard](https://dashboard.pantheon.io/sites/befd2142-355e-4527-bd5b-2b3a67210e9a) and click the **Multidev** tab. Then go to `Git branches` that will show the list of branches from which you can create a multidev environment. Once you find your `yalb-NNN` branch, trigger the multidev creation from there.
+- Once Pantheon has finished deploying your code, run the following commands in order:
+  - `terminus drush yale-integration-env.[environment] -- cr` to clear cache before updating references
+  - `terminus drush yale-integration-env.[environment] -- updb -y` to run database updates
+  - `terminus drush yale-integration-env.[environment] -- cr` to clear cache once more before importing configuration
+  - `terminus drush yale-integration-env.[environment] -- cim -y` to import configuration into DB
+  - `terminus drush yale-integration-env.[environment] -- cr` to lastly clear cache after configuration has been imported
+- You can now run ``terminus drush yale-integration-env.[environment] -- uli` to login into your desired environment.
```
Comment (vinmassaro) [inline README.md:24]: Looks like there is an extra backtick here.

Comment (nJim) [reply README.md:24]: Thanks for the extra set of eyes. I will fix this now. We can double-check everything tomorrow at Demo.

**Comment by nJim** [comment]
Marking this as approved. The work itself will have a very sort life in this form. These commands in the readme file are going to move into automated jobs in the next set of tickets.

**Pull request merged**